internal/instanceid/instanceid.go

```diff
@@ -6,17 +6,21 @@ import (
 	"github.com/google/uuid"
 )
 
+func Make() string {
```

Comment:
```suggestion
func New() string {
```

This feels more in line with how we normally do stuff, though happy to be educated if this is part of some convention.

Reply:
New relates to the builtin new() func that returns a pointer to a type. Make relates to the make() builtin which returns an instance of a complex type.